package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@duckduckgo/autoconsent": "^10.15.0",
-    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#13.1.0",
+    "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#8462e3b2d03015246e06ca5b6cfe9e381f626095",
```

Comment:
TODO: Pending to target specific release version

Reply:
This can be done now

Reply:
Done now; updated to `14.0.0`